dl/LSTM/requirements.txt

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
Hey @LakhanMardi, there are 135 packages in the requirements.txt file. Not all of them are getting used. Please remove all the unwanted packages. Thankyou.

Reply:
Sure. Will do it.